.github/ISSUE TEMPLATES/release-packages.md

```diff
@@ -0,0 +1,49 @@
+---
+name: Release some packages
+about: Use this template for tracking package releases.
+title: "Release some packages"
+---
+
+### Pending issues/pull-requests to be integrated to enable the release of packages:
+
+<!-- Mention below all issues/PRs that need to be integrated for the release,
+after consulting with the package maintainers. This is to help the release
+engineer track various things that the progress of release-work depends on -->
+
+- [ ] [Link](#)
+- [ ] [Link](#)
+
+-----
```

Comment:
I think this section makes more sense for a template in the repo that depends on cardano-base, eg. in ledger we could add a section like that: integration needed to account for releases from cardano-base and plutus.

The reason why I am saying that is because once release from cardabo-base is done and all of the check boxes below have been ticket it should be possible to close the ticket.

```suggestion
```

Reply:
I fully agree @lehins . :+1: . I was partly also thinking about the case where ongoing development branches/PRs sometimes need to be integrated but because the release process has already begun the release-engineer might need to keep a note about the several "to be merged for the release" kind of open issues. But this is highly unlikely in the case of `cardano-base` perhaps. Will take this out from here and only add this section in any downstream issues. :+1: